src/qt/sendcoinsdialog.cpp

```diff
@@ -757,10 +757,6 @@ void SendCoinsDialog::processSendCoinsReturn(const WalletModel::SendCoinsReturn
     case WalletModel::AbsurdFee:
         msgParams.first = tr("A fee higher than %1 is considered an absurdly high fee.").arg(BitcoinUnits::formatWithUnit(model->getOptionsModel()->getDisplayUnit(), model->wallet().getDefaultMaxTxFee()));
         break;
-    case WalletModel::PaymentRequestExpired:
-        msgParams.first = tr("Payment request expired.");
```

Comment:
Should the corresponding entries from the locae files also be removed, or is that done in Transifex somehow?

Reply:
That is responsibility of the [translation process](https://github.com/bitcoin/bitcoin/blob/master/doc/translation_process.md).